packages/account/src/Constants/auth-status-codes.ts

```diff
@@ -1,6 +1,6 @@
-import { TVerificationStatus } from '../../../../Types';
+import { TVerificationStatus } from '../Types';
 
-export const poa_status_codes: TVerificationStatus = {
+export const AUTH_STATUS_CODES: TVerificationStatus = {
     none: 'none',
```

Comment:
isnt better we make the key in capital letters?
 NONE: 'none',

Reply:
I tried that and the Type didn't allow capital letter keys
WIll move to shared and update the type as well